site/_data/config/bk_server.yaml

```diff
@@ -652,6 +652,9 @@ groups:
 
       Set this to 0 to disable the periodic check completely. Note that periodic checking will put extra load on the cluster, so it should not be run more frequently than once a day.
     default: 604800
+  - param: auditorPeriodicPlacementPolicyCheckInterval
+    description: The time interval between auditor placement policy checks, in seconds. The auditor placement policy check validates if the ensemble of segments of all the closed ledgers is adhering to the placement policy. It is just monitoring scrutiny but doesn't take any corrective measure other than logging error and reporting metrics. The default is once for every 30 hours.
+    default: 108000
```

Comment:
Very last question:
 Shouldn't we keep this feature disabled by default?

Reply:
valid point. will disable by default. will send the quick fix.